src/ajax-adapter.js

```diff
@@ -142,6 +146,7 @@ export class AjaxAdapter {
       },
       response => {
         var responseInput = new HttpResponse(response, requestInfo.zConfig);
+        if (!response.json) { responseInput.error = response; requestInfo.error(responseInput); return; }
```

Comment:
heya @jdanyow the fetch client will return an Error object, instead of a response, when there was a protocol error, like couldn't reach the server (server down, client internet connection down, etc).
So there is no response (no status, no statustext, no data), just an error object. As such there's no json() nor text() either :)